### DISCUSSION TITLE
Xdrip + Libre3

### DISCUSSION THREAD

**Discussion by Kustomwireless** [NightscoutFoundation/xDrip#3538] [Q&A]
### Discussed in https://github.com/NightscoutFoundation/xDrip/discussions/3536

<div type='discussions-op-text'>

<sup>Originally posted by **Kustomwireless** June 14, 2024</sup>
Can someone explain why Xdrip + is not able to work direct with Libre 3 as some of the other Apps do such as Juggluco??
 Some back round   We have been using Xdrip + for about 8 years with US Libre and Blucon also bubble.

For Now We are using Juggluco to send data to Xdrip + and it seems to be working pretty well. With one exception and that is the sensors seem to stop working 2 to 3 days early consistently don't know if there is a connection or not!
 Just looking to simplify  things for my wife. Is there any other way accomplish  what we are doing?? I also follow her with Xdrip.

</div>

**Comment by Navid200** [comment]
Do not open an issue.
Instructions clearly explain that.
